testhelper/docker/resource/transformer/transformer.go

```diff
@@ -0,0 +1,94 @@
+package transformer
+
+import (
+	"errors"
+	"fmt"
+	"net/http"
+
+	"github.com/ory/dockertest/v3"
+	"github.com/ory/dockertest/v3/docker"
+
+	"github.com/rudderlabs/rudder-go-kit/httputil"
+	"github.com/rudderlabs/rudder-go-kit/testhelper/docker/resource"
+)
+
+type Resource struct {
+	TransformerURL string
+	Port           string
+}
+
+type config struct {
+	Repository   string
+	Tag          string
+	ExposedPorts []string
+	Env          []string
+}
+
+func WithConfigBackendURL(url string) func(*config) {
```

Comment:
Can we have also an option for passing a custom tag (i.e. docker image version)?